integration/test-integration/src/test/resources/openAPIs/api_policy_openAPI.yaml

```diff
@@ -56,18 +62,6 @@ paths:
       responses:
         '200':
           description: successful operation
-  /echo-full/rewrite-policy-with-capture-groups-invalid-param/shops/{shopId}/pets/{petId}/orders:
-    get:
-      summary: rewrite based policies appied with capture groups with invalid params
-      responses:
-        '200':
-          description: successful operation
-  /echo-full/rewrite-policy-with-capture-groups-invalid-chars/shops/{shopId}/pets/{petId}/orders:
-    get:
-      summary: rewrite based policies appied with capture groups with invalid chars
-      responses:
-        '200':
-          description: successful operation
```

Comment:
This is removed because now, the whole API deployment fails if one policy is invalid right?

Reply:
Yes, that is the reason.